packages/flutter_test/lib/src/web.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#comment-all-ignores

Also, will what is defined in here show up on api.flutter.dev? It should have docs then.

Reply:
This is basically the same situation as https://github.com/flutter/flutter/blob/18340ea16cee656f14daa2aa686f1e05be8f7daf/packages/flutter/lib/src/web.dart#L5. I'll go ahead and add the same comment to the ignore.

Reply:
These should not show up on api.flutter.dev. If they do, there's probably a bug somewhere.